jtelegrambotapi-core/src/main/java/com/jtelegram/api/commands/filters/CommandFilter.java

```diff
@@ -0,0 +1,30 @@
+package com.jtelegram.api.commands.filters;
+
+import com.jtelegram.api.commands.Command;
+import com.jtelegram.api.events.message.TextMessageEvent;
+
+/**
+ * @author Nick Robson
```

Comment:
We're not allowing these, not to take away from your contributions but because this class will probably be modified by others in the future and this takes away from their contributions.